code/modules/projectiles/projectile/beams.dm

```diff
@@ -144,6 +144,34 @@
 /obj/projectile/beam/emitter/singularity_pull()
 	return //don't want the emitters to miss
 
+//This is basically a cannon beam
+/obj/projectile/beam/emitter/heavy
+	name = "heavy emitter beam"
+	icon_state = "emitter_heavy"
+	damage = 600
```

Comment:
behold, perry the platypus! the megafauna snipe-inator!

Reply:
also, uh, does this use the normal emitter beam range?

what IS the normal emitter beam range?

Reply:
50 tiles i think